CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ message(STATUS "CCF release version=${CCF_RELEASE_VERSION}")
 #
 if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
   set(CMAKE_INSTALL_PREFIX
-      "/opt/ccf/ccf-${CCF_VERSION}"
+      "/opt/ccf/ccf-${CCF_RELEASE_VERSION}"
```

Comment:
In the case of a "between releases" commit, I think it would still be nice to have a suffix that doesn't match the previous release. E.g. if we are currently 5 commits after 0.11, it would be nice if the install path was something like `ccf-0.11-5-g<sha>` rather than `ccf-0.11`, which would override the latest release.

Reply:
Sure